src/ClusterBootstrap/gs_tools.py

```diff
@@ -347,13 +347,13 @@ def create_storage_with_config( configGrp, location ):
     storagename = configGrp["name"] + location
     output = create_storage_account( storagename, configGrp["sku"], location)
     if verbose: 
-        print ( "Storage account %s" % output )
+        print(( "Storage account %s" % output ))
```

Comment:
double brackets

Reply:
Yes, I see many of this. I think it's because 2to3 will add brace whenever it encountered a `print`

Reply:
Will delete these